lib/snapshot/runner.rb

```diff
@@ -14,7 +14,7 @@ def work(clean: true, build: true)
       counter = 0
       errors = []
 
-      FileUtils.rm_rf SnapshotConfig.shared_instance.screenshots_path if SnapshotConfig.shared_instance.clear_previous_screenshots
+      FileUtils.rm_rf SnapshotConfig.shared_instance.screenshots_path if SnapshotConfig.shared_instance.clear_previous_screenshots and take_snapshots
```

Comment:
This will not work as expected, you'l have to add `()` around it.

```ruby
if (SnapshotConfig.shared_instance.clear_previous_screenshots and take_snapshots)
```